src/js/apps/themes.jsx

```diff
@@ -0,0 +1,19 @@
+export const defaultTheme = {
```

Comment:
I suggest add color codes to constant in constant.js ?

Reply:
I don't know if this is really necessary, those color codes are only used here, and exporting them from constats.js will only add an unnecessary layer.  I would consider that if those color codes where used anywhere else tough. 
What do you think?